app/scripts/modules/amazon/src/serverGroup/details/advancedSettings/AdvancedSettingsView.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import autoBindMethods from 'class-autobind-decorator';
+import { IAmazonServerGroupView } from 'amazon/domain';
+
+export interface IAdvancedSettingsViewProps {
```

Comment:
Since we will have multiple `*AdvancedSettingsView`s, maybe be more specific with the naming of these views? `ServerGroupAdvancedSettingsView` a little long winded, but then won't collide...